### PR TITLE
Feature staticssr support + escaping string

### PR DESCRIPTION
**Pull request by jamesjjk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .vscode
-build
```
Comment (tizmagik) [inline .gitignore]: Any reason to include the build directory?

File: src/HeadTag.js [hunk 2]
```diff
@@ -36,15 +36,23 @@ export default class HeadTag extends Component {
   render() {
     const { tag: Tag, ...rest } = this.props;
 
-    const Comp = <Tag {...rest} />;
-
     if (this.state.canUseDOM) {
+      const Comp = (
+        <Tag
+          key={`${Tag}${Object.keys(rest)
```
Comment (tizmagik) [inline src/HeadTag.js]: Help me understand why do we need a key field?

File: src/HeadTag.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ export default class HeadTag extends Component {
     this.setState({ canUseDOM: true });
 
     const { tag, children, ...rest } = this.props; // eslint-disable-line react/prop-types
-    const ssrTags = document.head.querySelector(`${tag}${buildSelector(rest)}[data-reactroot=""]`);
+    const ssrTags = document.head.querySelector(`${tag}${buildSelector(rest)}[data-ssr=""]`);
```
Comment (tizmagik) [inline src/HeadTag.js]: What do you think about naming this `data-rh` instead (react-head)?

File: src/buildSelector.js [hunk 4]
```diff
@@ -1,6 +1,10 @@
+/* eslint no-useless-escape:0 */
 const buildSelector = obj =>
   Object.keys(obj)
-    .map(k => `[${k}="${obj[k]}"]`)
+    .map(k => {
+      const cnt = String(obj[k]).replace(/["\\]/g, '\\$&');
```
Comment (tizmagik) [inline src/buildSelector.js]: Can you add a test case for this please? (Where the escape is necessary)

**Comment by tizmagik** [comment]
Hey @jamesjjk lmk if you're still interested in closing the loop on this. If not, I can merge your current state into another branch and I can finish it up for you. I'd like to try and get a release out today or tomorrow 😁 

Thanks again for this, huge help!

File: .gitignore (hunk 1 above)
Comment (jamesjjk) [reply .gitignore]: Only needed it when using my version in npm

File: src/HeadTag.js (hunk 3 above)
Comment (jamesjjk) [reply src/HeadTag.js]: feel free (remember to duplicate change to the render method)

File: src/HeadTag.js (hunk 2 above)
Comment (jamesjjk) [reply src/HeadTag.js]: As far as I'm concerned they are only needed for SSR support.

**Comment by jamesjjk** [comment]
@tizmagik Feel free to merge, the Keys are required for SSR and the headcollector, else you will get an error.

File: src/buildSelector.js (hunk 4 above)
Comment (tizmagik) [reply src/buildSelector.js]: @jamesjjk do you have an example/use case you could share for this?

**Pull request merged**